bindata/manifests/operator-webhook/server.yaml

```diff
@@ -47,6 +47,12 @@ spec:
       - key: "node.kubernetes.io/not-ready"
         operator: Exists
         effect: NoSchedule
+      {{- if .ImagePullSecrets }}
```

Comment:
should we add also to NRI ? (bindata/manifests/webhook/)

Reply:
done